functions/Get-DbaConfiguration.ps1

```diff
@@ -0,0 +1,108 @@
+﻿FUNCTION Get-DbaConfiguration
```

Comment:
I based the name off of the proposed name from the official Microsoft board, which was likely based off of SMO. I'm good with keeping it. Alternatively, we can do what I did in a few other commands: Get-DbaSpConfigure 

Copy-SqlSpConfigure
Export-SqlSpConfigure
Import-SqlSpConfigure

Not a huge fan of Instance but have no discernible reason.